### PR TITLE
Will branch

### PR DESCRIPTION
**Pull request by wtruscott**
this is the current App.js and i worked on giving props to Form

### CODE REVIEW

File: src/component/Form.js [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
                 type="text"
                 id="artist"
+                placeholder="artist"
+                value={formData.artist}
+                onChange={handleChange}
             />
 
             <label htmlFor="duration">DURATION</label>
             <input 
                 type="string"
```
Comment (kinr-jay) [inline src/component/Form.js]: type="text"

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,61 @@
...
+const deleteSong = (song) => {
+fetch(url + "/tunes/" + song._id, {
+  method: "delete"
+})
+.then(() => {
+  getSongs()
+})
+}
   return (
-    <div className="App">
```
Comment (kinr-jay) [inline src/App.js:6]: keep App class div

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,61 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <main>
```
Comment (kinr-jay) [inline src/App.js]: ditch main element

**Review by kinr-jay — CHANGES_REQUESTED** [review]
See three comments for updates

**Review by kinr-jay — APPROVED** [review]
Thanks, Will!

**Review by wtruscott — COMMENTED** [review]
Changes have been made

**Pull request merged**